src/assets/scss/_record.scss

```diff
@@ -1 +1,144 @@
 // record component rules
+.top-panel-container {
+    padding-top: 20px;
```

Comment:
Please make sure you've pulled the latest `react` branch into this branch. If you've done this and you mimicked the same page structure as recordset (and master), then you don't need to manually set all these padding and margins. The [default rules in `app.scss`](https://github.com/informatics-isi-edu/chaise/blob/react/src/assets/scss/app.scss#L413) should take care of this.